ci/prow/config.yaml

```diff
@@ -245,8 +245,6 @@ presubmits:
         - "--postsubmit-gcs-bucket=knative-prow"
```

Comment:
This is the default value, can we remove it as well?

Reply:
Done.